lib/root/widgets/cart_icon.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:badges/badges.dart' as badges;
```

Comment:
I'm answering the comment from your other PR https://github.com/BASE1com/flex-storefront/pull/50#discussion_r1591558171 here 🙃 

Instead of prefixing, you can hide the `Badge` export from material:

`import 'package:flutter/material.dart' hide Badge;`